openslides/users/static/templates/users/group-detail.html

```diff
@@ -0,0 +1,14 @@
+<h1>{{ group.name }}</h1>
+
+<div id="submenu">
+  <a ui-sref="users.group.list" class="btn btn-sm btn-default">
+    <i class="fa fa-angle-double-left fa-lg"></i>
+    <translate>Back to overview
```

Comment:
I would like valid html5. For that reason I would like that you add the closing translation tag.

If you find a source, that custom tags can have optional closing tags, and that this translation tag is one of them, then it is ok for me to skip the closing tag.